server/tstCompletion.ts

```diff
@@ -165,24 +168,38 @@ export async function getTstCompletionData(
         "UNIT",
         completionData.position.line
       );
+
+      autocompletionParams.unit = unitName;
+
       // TBD will need to change how this is done during the fix for issue #170
       // we use python to get a list of subprograms by creating a fake VALUE line
       // with the unitName set to what we found
-      let choiceArray = ["<<INIT>>", "<<COMPOUND>>"];
-      let choiceKind = "Keyword";
+      // let choiceArray = ["<<INIT>>", "<<COMPOUND>>"];
+      let choiceKind = "";
+      let choiceArray: string[] = [];
       if (unitName.length > 0) {
         const choiceData = await getChoiceData(
           choiceKindType.choiceListTST,
           enviroPath,
-          "TEST.VALUE:" + unitName + "."
+          upperCaseLine,
+          autocompletionParams
         );
+        console.log("++++++++++++++++++++++++++++++");
+        console.log(choiceData);
+        console.log("++++++++++++++++++++++++++++++");
+        // const choiceData = await getChoiceData(
+        //   choiceKindType.choiceListTST,
+        //   enviroPath,
+        //   "TEST.VALUE:" + unitName + "."
+        // );
         returnData.extraText = choiceData.extraText;
         returnData.messages = choiceData.messages;
         choiceKind = choiceData.choiceKind;
         // append actual choices to the default INIT and COMPOUND
-        choiceArray = choiceArray.concat(choiceData.choiceList);
+        // choiceArray = choiceArray.concat(choiceData.choiceList);
         // <<GLOBAL>> is valid on VALUE lines but not as a function name!
-        choiceArray = filterArray(choiceArray, "<<GLOBAL>>");
+        // choiceArray = filterArray(choiceArray, "<<GLOBAL>>");
```

Comment:
Same

Reply:
Removed in 78bf51f72c4d2f1a11d3f97b4d1575e0ba69e717 .